LICENSE-MIT

```diff
@@ -0,0 +1,23 @@
+Permission is hereby granted, free of charge, to any
```

Comment:
There usually is a header outlining and attributing the author: https://spdx.org/licenses/MIT.html

I don't know where this file is from, but it appears the authors did not want to deal with updating the author information or similar.

Reply:
> I don't know where this file is from

As mentioned in https://github.com/AudioSceneDescriptionFormat/asdf-rust/issues/37#issuecomment-2033705540, I copied the files from the Rust project.

> it appears the authors did not want to deal with updating the author information or similar.

I also mentioned that discussions are here: https://github.com/rust-lang/rust/pull/43498

Those go into details about the motivations.

Reply:
Okay. I mean, it's up to you and not a blocker for merging this. :ok_hand: